### ISSUE TITLE
CSS in JS support

### ISSUE THREAD

**Issue by LionyxML** [thefrontside/frontmacs#159]
Hello there!

Please keep an eye on this repo:
https://github.com/orzechowskid/tsx-mode.el

What I though interesting: it implements CSS-IN-JS, which is HIGHLY desirable, since styled components are becoming huge on React community and regular configs treats what is inside back-ticks as regular strings.

**Comment by cowboyd** [comment]
Thanks for the heads up! I'll definitely keep an eye on it. It's a lot of dependencies to pull in, but the current hack of using web-mode is not the best either.

**Comment by LionyxML** [comment]
Yeah, it is in the early stage but who knows what it might become =)

One small thing you might know (and it is ok if you don't), I am on MacOS with a precompiled version of emacs.

I find this tooltips SO damn small.
![image](https://user-images.githubusercontent.com/16169950/159345207-cb2631e9-0976-40bf-a477-72f80331c4ce.png)

I tried to mess with tooltip fonts but this seems not to be affected by my configs.
